client/src/components/Workflow/Editor/Lint.vue

```diff
@@ -1,470 +1,201 @@
-                    </span>
-                </LintSection>
-                <!--
-                Had tried to make this a middle panel thing, but it breaks the workflow
-                to modify it while the canvas is not displayed.
-                <b-button
-                    id="done"
-                    type="submit"
-                    variant="primary"
-                    v-b-tooltip.bottom.hover
-                    title="Return to the workflow editor"
-                    @click="doReturn"
-                >
-                    <i class="icon fa fa-edit" /> Return to Editor
-                </b-button>
-                -->
-            </b-col>
-        </b-row>
-    </b-container>
+    <b-card header-tag="header" body-class="p-0">
+        <template #header>
+            <div class="mb-1 font-weight-bold">
+                <font-awesome-icon icon="magic" class="mr-1" />
+                Best Practices Review
+            </div>
+            <div v-if="showRefactor">
+                <a href="#" @click="onRefactor">
+                    Try to automatically fix issues.
+                </a>
+            </div>
+        </template>
+        <b-card-body>
+            <LintSection
+                :okay="checkAnnotation"
+                success-message="This workflow is annotated. Ideally, this helps the executors of the workflow
+                    understand the purpose and usage of the workflow."
+                warning-message="This workflow is not annotated. Providing an annotation helps workflow executors
+                    understand the purpose and usage of the workflow."
+                attribute-link="Annotate your Workflow."
+                @onClick="onAttributes"
+            />
+            <LintSection
+                :okay="checkCreator"
+                success-message="This workflow defines creator information."
+                warning-message="This workflow does not specify creator(s). This is important metadata for workflows
+                    that will be published and/or shared to help workflow executors know how to cite the
+                    workflow authors."
+                attribute-link="Provide Creator Details."
+                @onClick="onAttributes"
+            />
+            <LintSection
+                :okay="checkLicense"
+                success-message="This workflow defines a license."
+                warning-message="This workflow does not specify a license. This is important metadata for workflows
+                    that will be published and/or shared to help workflow executors understand how it
+                    may be used."
+                attribute-link="Specify a License."
+                @onClick="onAttributes"
+            />
+            <LintSection
+                success-message="Workflow parameters are using formal inputs."
+                warning-message="This workflow uses implicit workflow parameters. They should be replaced with
```

Comment:
Again, I don't think there's anything implicit about them. An alternative would by untyped, but that seems too technical.

Reply:
Renamed it to untyped parameters internally, we can name it whatever we we like in the ui.